gammapy/utils/tests/test_energy.py

```diff
@@ -6,14 +6,14 @@
 from astropy.io import fits
 from ...utils.testing import requires_data
 from ...datasets import gammapy_extra
-from ...spectrum import Energy, EnergyBounds
+from ...utils.energy import Energy, EnergyBounds
 
 
 def test_Energy():
     # Explicit constructor call
     energy = Energy([1, 3, 6, 8, 12], 'TeV')
     actual = type(energy).__module__
-    desired = 'gammapy.spectrum.energy'
+    desired = 'gammapy.utils.energy'
```

Comment:
What's the point of testing for the module the class is located in?
Maybe remove this assert?
Or if something like this is needed, maybe assert on the instance type or class name instead?